server.py

```diff
@@ -13,54 +13,106 @@
 apm = ElasticAPM(app, logging=True)
 cors = CORS(app, resources={r"/*": {"origins": "*"}})
 
-"""Request health status of the api"""
+def __is_session_valid(email, session_token):
+    if email and session_token:
+        response = betaAuth.check_session(email, session_token)
+        if response['session'] == True:
```

Comment:
If providing an invalid (here spoiled) session token, `response['session']` throws a `KeyError`, which leads to a `500` server response:

![bildschirmfoto vom 2018-11-01 18-12-30](https://user-images.githubusercontent.com/15818147/47867430-64a61000-ddf9-11e8-8630-d6dbc31e8240.png)

![bildschirmfoto vom 2018-11-01 18-11-44](https://user-images.githubusercontent.com/15818147/47867436-67086a00-ddf9-11e8-8b11-1efe756ab37f.png)

The problematic code for this might reside in `beatAuth.check_session`.